### PR TITLE
feat: growth marketing manager role to jobs page

### PR DESCRIPTION
**Pull request by marneysof**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: src/markdown/jobs/growthmarketing [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+### Overview
+
+- **Position Type:** Full Time
+- **Location**: Munich, Germany
+- **Components**: HubSpot, Google Analytics
+
+### What You'll Do
+
+- Build programs and digital campaigns to help people understand, try, and adopt Ory products. 
+- Optimize through experimentation. Create A/B tests to iterate on promotional content to increase engagement and product adoption.
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/growthmarketing:37]:
```suggestion
- Optimize through experimentation. Create A/B tests to iterate on content to increase engagement and product adoption. 
```

File: src/markdown/jobs/growthmarketing [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- **Location**: Munich, Germany
+- **Components**: HubSpot, Google Analytics
+
+### What You'll Do
+
+- Build programs and digital campaigns to help people understand, try, and adopt Ory products. 
+- Optimize through experimentation. Create A/B tests to iterate on promotional content to increase engagement and product adoption. 
+- Distribute Ory Corp core messaging across digital channels. Create and expand digital channels to engage prospects and customers.
+- Build and adjust marketing operations infrastructure that incorporates performance measurements and analytics. 
+- Build retargeting campaigns and perform segmentation analysis for personalized conversion journeys across different groups of users.
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/growthmarketing:40]:
```suggestion
- Build campaigns (incl. retargeting) and perform segmentation analysis for personalized conversion journeys across different groups of users.
```

File: src/markdown/jobs/growthmarketing [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+- Collaborate with team members (and perhaps an external agency) to execute on SEO priorities to increase organic traffic to Ory Corp and associated websites in our product portfolio.
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/growthmarketing:41]:
```suggestion
- Collaborate with team members (and perhaps an external agency) to execute on SEO priorities to increase organic traffic to Ory Corp and associated websites.
```

File: src/markdown/jobs/growthmarketing [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+- Collaborate with other departments (HR, Sales, Engineering) to maximize yield from shared channels and programs.
+- Share key insights from experiments and projects.
+
+### We’re looking for someone who has
+
+- Familiarity with modern digital growth strategies including SEO, SEM, social ads, content syndication, and more
+- Experience leading cross-department projects and collaborating with web developers to drive meaningful impact
+- Experience working with technical product teams (software, security, or cloud infrastructure)
+- Strong analytical skills with an ability to compile and analyze campaign data to inform decisions
+- Experience generating demand at an enterprise B2B company
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/growthmarketing:51]:
```suggestion
- Ideally, experience generating demand for product-focused SaaS enterprise in the B2B market.
```

File: src/markdown/jobs/growthmarketing [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+- Experience working closely with product marketing managers to amplify key content and features
+- Strong editorial skills
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/growthmarketing:53]:
```suggestion
- Strong editorial skills in English
```

File: src/markdown/jobs/growthmarketing [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+- Entrepreneurial zest and a hands-on attitude
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/growthmarketing:54]:
```suggestion
- Entrepreneurial mind set and a hands-on attitude
```

File: src/markdown/jobs/growthmarketing [hunk 7]
```diff
@@ -0,0 +1,61 @@
+---
+published: true
+path: '/jobs/growthmarketingmanager/'
+title: 'Growth Marketing Manager'
+metaTitle: 'Open Marketing Careers at Ory'
+metaDescription: 'Read the Growth Marketing Manager job description.'
+lastUpdatedAt: 'September 30, 2021'
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/growthmarketing:7]:
```suggestion
lastUpdatedAt: 'December 13, 2021'
```

**Review by BlueAcidFrog — DISMISSED** [review]

**Comment by aeneasr** [comment]
Good to go @BlueAcidFrog ?

File: src/markdown/jobs/growthmarketing [hunk 8]
```diff
@@ -0,0 +1,61 @@
+---
+published: true
+path: '/jobs/growthmarketingmanager/'
+title: 'Growth Marketing Manager'
+metaTitle: 'Open Marketing Careers at Ory'
+metaDescription: 'Read the Growth Marketing Manager job description.'
+lastUpdatedAt: 'December 13, 2021'
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/growthmarketing:7]:
```suggestion
lastUpdatedAt: 'December 15, 2021'
```

**Review by BlueAcidFrog — APPROVED** [review]

File: src/markdown/jobs/growthmarketing [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+- Build and adjust marketing operations infrastructure that incorporates performance measurements and analytics. 
+- Build campaigns (incl. retargeting) and perform segmentation analysis for personalized conversion journeys across different groups of users.
+- Collaborate with team members (and perhaps an external agency) to execute on SEO priorities to increase organic traffic to Ory Corp and associated websites.
+- Collaborate with other departments (HR, Sales, Engineering) to maximize yield from shared channels and programs.
+- Share key insights from experiments and projects.
+
+### We’re looking for someone who has
+
+- Familiarity with modern digital growth strategies including SEO, SEM, social ads, content syndication, and more
+- Experience leading cross-department projects and collaborating with web developers to drive meaningful impact
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/growthmarketing:48]:
```suggestion
- Several years of experience leading cross-department projects and collaborating with web developers to drive meaningful impact
```

File: src/markdown/jobs/growthmarketing [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+- Experience working with technical product teams (software, security, or cloud infrastructure)
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/growthmarketing:49]:
```suggestion
- Several years of experience working with technical product teams (software, security, or cloud infrastructure)
```

File: src/markdown/jobs/growthmarketing [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+- Strong analytical skills with an ability to compile and analyze campaign data to inform decisions
+- Ideally, experience generating demand for product-focused SaaS enterprise in the B2B market.
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/growthmarketing:51]:
```suggestion
- Experience in successfully generating demand for product-focused SaaS enterprises in the B2B market.
```

File: src/markdown/jobs/growthmarketing [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+- Experience working closely with product marketing managers to amplify key content and features
+- Strong editorial skills in English
+- Entrepreneurial mind set and a hands-on attitude
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/growthmarketing:54]:
```suggestion
- Entrepreneurial mindset and very important, hands-on attitude
```

**Review by BlueAcidFrog — DISMISSED** [review]

**Pull request merged**